org-roam.el

```diff
@@ -251,12 +259,17 @@ it as FILE-PATH."
                (link-type (cond ((and (string= type "file")
                                       (org-roam--org-file-p path))
                                  "roam")
```

Comment:
might want to change this away from "roam" into something like "org", just to prevent confusion down the road.

Reply:
I'm not sure I understand the change you're suggesting

Reply:
Updated